aws/table_aws_route53_vpc_association_authorization.go

```
@@ -0,0 +1,134 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsRoute53VPCAssociationAuthorization(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_route53_vpc_association_authorization",
+		Description: "AWS Route53 VPC Association Authorization",
+		List: &plugin.ListConfig{
+			ParentHydrate: listHostedZones,
+			Hydrate:       listVPCAssociationAuthorization,
```

Please add the ignore error config for the API such as:

```
IgnoreConfig: &plugin.IgnoreConfig{
	ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"NoSuchHostedZone"}),
},
```